### PR TITLE
docs: Update uninstall to ensure CRDs are deleted

### PR DESCRIPTION
**Pull request by david-yu**
Sometimes CRDs are not deleted since the finalizers don't fully complete execution. The added commands ensure that CRDs are deleted during the uninstall process. 

Sequence of operations

```
❯ export HELM_RELEASE=dyu
helm install ${HELM_RELEASE} hashicorp/consul --version "0.39.0" --create-namespace -n consul -f ./server-config.yaml
NAME: dyu
LAST DEPLOYED: Tue Jan 11 12:12:48 2022
NAMESPACE: consul
STATUS: deployed
REVISION: 1
NOTES:
Thank you for installing HashiCorp Consul!

Now that you have deployed Consul, you should look over the docs on using
Consul with Kubernetes available here:

https://www.consul.io/docs/platform/k8s/index.html

Your release is named dyu.

To learn more about the release, run:

  $ helm status dyu
  $ helm get all dyu

~/projects/consul1.11 1m 2s
❯ kubectl create ns consul
kubectl create secret -n consul generic license --from-file=key=./consul.hclic

~/projects/consul1.11
❯ k apply -f service-defaults.yaml
servicedefaults.consul.hashicorp.com/foo created
servicedefaults.consul.hashicorp.com/foo2 created

~/projects/consul1.11
❯ kubectl delete crd --selector app=consul
customresourcedefinition.apiextensions.k8s.io "exportedservices.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "ingressgateways.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "meshes.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "proxydefaults.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "servicedefaults.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "serviceintentions.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "serviceresolvers.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "servicerouters.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "servicesplitters.consul.hashicorp.com" deleted
customresourcedefinition.apiextensions.k8s.io "terminatinggateways.consul.hashicorp.com" deleted

❯ k get crd
NAME                                             CREATED AT
backendconfigs.cloud.google.com                  2022-01-11T20:08:24Z
capacityrequests.internal.autoscaling.gke.io     2022-01-11T20:07:59Z
frontendconfigs.networking.gke.io                2022-01-11T20:08:25Z
managedcertificates.networking.gke.io            2022-01-11T20:08:01Z
serviceattachments.networking.gke.io             2022-01-11T20:08:27Z
servicenetworkendpointgroups.networking.gke.io   2022-01-11T20:08:25Z
storagestates.migration.k8s.io                   2022-01-11T20:08:04Z
storageversionmigrations.migration.k8s.io        2022-01-11T20:08:04Z
updateinfos.nodemanagement.gke.io                2022-01-11T20:08:06Z
volumesnapshotclasses.snapshot.storage.k8s.io    2022-01-11T20:08:03Z
volumesnapshotcontents.snapshot.storage.k8s.io   2022-01-11T20:08:03Z
volumesnapshots.snapshot.storage.k8s.io          2022-01-11T20:08:03Z

~/projects/consul1.11
❯ helm uninstall ${HELM_RELEASE} -n consul
release "dyu" uninstalled
```

After uninstall and re-installing Consul K8s via the Helm uninstall steps I get the following: 

```
> export HELM_RELEASE=dyu
helm install ${HELM_RELEASE} hashicorp/consul --version "0.39.0" --create-namespace -n consul -f ./server-config.yaml

~/projects/consul1.11
❯ k get servicedefaults -n consul
No resources found in consul namespace.

~/projects/consul1.11
❯ k get crd
NAME                                             CREATED AT
backendconfigs.cloud.google.com                  2022-01-11T20:08:24Z
capacityrequests.internal.autoscaling.gke.io     2022-01-11T20:07:59Z
exportedservices.consul.hashicorp.com            2022-01-11T20:30:59Z
frontendconfigs.networking.gke.io                2022-01-11T20:08:25Z
ingressgateways.consul.hashicorp.com             2022-01-11T20:31:00Z
managedcertificates.networking.gke.io            2022-01-11T20:08:01Z
meshes.consul.hashicorp.com                      2022-01-11T20:31:00Z
proxydefaults.consul.hashicorp.com               2022-01-11T20:31:00Z
serviceattachments.networking.gke.io             2022-01-11T20:08:27Z
servicedefaults.consul.hashicorp.com             2022-01-11T20:31:00Z
serviceintentions.consul.hashicorp.com           2022-01-11T20:31:00Z
servicenetworkendpointgroups.networking.gke.io   2022-01-11T20:08:25Z
serviceresolvers.consul.hashicorp.com            2022-01-11T20:31:00Z
servicerouters.consul.hashicorp.com              2022-01-11T20:31:00Z
servicesplitters.consul.hashicorp.com            2022-01-11T20:30:59Z
storagestates.migration.k8s.io                   2022-01-11T20:08:04Z
storageversionmigrations.migration.k8s.io        2022-01-11T20:08:04Z
terminatinggateways.consul.hashicorp.com         2022-01-11T20:30:59Z
updateinfos.nodemanagement.gke.io                2022-01-11T20:08:06Z
volumesnapshotclasses.snapshot.storage.k8s.io    2022-01-11T20:08:03Z
volumesnapshotcontents.snapshot.storage.k8s.io   2022-01-11T20:08:03Z
volumesnapshots.snapshot.storage.k8s.io          2022-01-11T20:08:03Z
```

### CODE REVIEW

File: website/content/docs/k8s/operations/uninstall.mdx [hunk 1]
```diff
@@ -42,6 +42,13 @@ Run the `helm uninstall` **and** manually remove resources that Helm does not de
    $ helm uninstall consul
    release "consul" uninstalled
    ```
+   
+   
+1. Ensure that the CRDs were previously created for Consul on Kubernetes are deleted, so subsequent installs of Consul on Kubernetes on the same Kubernetes cluster do not get blocked. 
+
+   ```shell-session
+   $ kubectl get crd | grep consul.hashicorp.com | awk '{print $1}' | xargs kubectl delete crd
```
Comment (blake) [inline website/content/docs/k8s/operations/uninstall.mdx:50]:
```suggestion
   $ kubectl delete crd --selector app=consul
```
This is a simpler solution.

**Review by thisisnotashwin — APPROVED** [review]
nice work @david-yu

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/549036.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit d20230fac1e89678ba6f5e26bad4d2fff99fe9f2 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit d20230fac1e89678ba6f5e26bad4d2fff99fe9f2 onto `release/1.11.x` succeeded!

**Pull request merged**